lib/Optimizer/Builder/Factory.cpp

```diff
@@ -73,7 +73,7 @@ cudaq::cc::StructType factory::buildInvokeStructType(FunctionType funcTy) {
     eleTys.push_back(genBufferType</*isOutput=*/false>(inTy));
   for (auto outTy : funcTy.getResults())
     eleTys.push_back(genBufferType</*isOutput=*/true>(outTy));
-  return cudaq::cc::StructType::get(ctx, eleTys /*packed=false*/);
+  return cudaq::cc::StructType::get(ctx, eleTys /*isPacked = false*/);
```

Comment:
```suggestion
  return cudaq::cc::StructType::get(ctx, eleTys);
```
I think we're ok to just remove this comment altogether here. Typically, the convention is to use `/*flagName=*/true` when a `true` or `false` is required. Just having magic values appear as arguments is discouraged.